---
banner:
  enable: true
  bg_image: images/slider-bg.jpg
  bg_overlay: true
  title: Immunogenomics Data Analysis Working Group
  content: "<div align=\"left\">\n<font size=\"+1\">\nIDAWG is an international collaboration
    of histocompatibility and immunogenetics investigators who share the goal of facilitating
    the sharing of immunogenomic data (HLA, KIR, etc.) and fostering the consistent
    analysis and interpretation of those data by the immunogenomics community and
    the larger genomics communities.\n<br />\n<br />\nThe working group was formed
    in advance of the 16th International HLA and Immunogenetics Workshop (IHIW) and
    Conference with the intent to present its recommendations on topics of data-management
    and data-analysis at the 16th IHIW and Conference in 2012. \n</font/>\n</div>"
  button:
    enable: false
    label: Discover Our Project
    link: project/
about:
  enable: true
  title: About Us
  description: |-
    The IDAWG is an international collaboration of histocompatibility and immunogenetics investigators who share the goal of making immunogenomic data (HLA, KIR, etc.) more useful across studies and between different groups.
    <br />
    Toward these ends, the working group will develop methods, standards, tools and recommendations intended to:
    <br>
    <br>
  content: |
    <ol>
      <li>Record, store and transmit immunogenomic data without obscuring the limitations of the typing method used, allow easy identification of allelic equivalency under successive nomenclatures, make data both human-readable (e.g., flat-text file) and machine-readable (e.g., XML file), conform to extant nomenclature rules, all without the use of proprietary platforms.</li>
    <br />
      <li>Document ambiguity reduction methods used, permit reproducible ambiguity reduction, and permit equivalency under different methods of ambiguity reduction.</li>
    <br />
      <li>Foster portability between extant analysis tools and methods to permit maximum access to investigators (e.g., web-based tools).</li>
    <br />
      <li>Encourage consistent data formats in future analytical methods, promoting widespread accessibility and application.</li>
    <br />
      <li>Foster methodological consistency in the analysis of low frequency alleles and heterogeneous data, the estimation of haplotypes, Hardy-Weinberg testing of highly polymorphic data, the application of measures of and adjustment for linkage disequilibrium, tests for selection and measures of population differentiation, the calculation of odds ratios, relative risks, etc., corrections for multiple testing, mitigation of false positive readings.</li>
    <br />
      <li>Develop novel methods of data analysis for highly polymorphic loci in disease association and population studies (e.g., peptide and nucleotide-level analyses, multidimensional scaling analyses, and neural network analyses). </li>
     </ol>
  image: images/wrapper-img.png
portfolio:
  enable: true
  bg_image: images/feature-bg.jpg
  title: PROJECT RATIONALE
  content: "Immunogenetics is an international field with a history that spans more
    than fifty years. Immunogenetic genotyping, data-generation and data-analysis
    methods have proliferated in this time, in many cases in inconsistent ways. We
    have identified five areas where (often unavoidable) inconsistency can be introduced
    into genotype data; these inconsistencies in turn contribute to variation in the
    analysis of immunogenomic data, and the interpretation of such analyses.\n<br>
    \n<br> \n<font size=\"+1\"><p style=\"color:black;\"><b>CHALLENGES TO CONSISTENCY
    IN IMMUNOGENETIC DATA MANAGEMENT AND ANALYSIS</b></p></font>\n<i>Variation in
    Typing Methodology</i>\n<br> \nThe large variety of typing methods in use results
    in datasets being generated with varying levels of resolution, so that a particular
    allele (or set of equivalent alleles) may be identified in a sample using one
    method, while a slightly different allele (or set of equivalent alleles) may be
    identified in the same sample, using a different method.\n<br> \n<br> \n<i>Changes
    in Nomenclature</i>\n<br> \nThe continual evolution of the nomenclature conventions
    over time has resulted in a progression of allele and locus identifiers that are
    not always easily inter-related. It is usually the case that the nomenclature
    used to identify the alleles in a dataset is \"frozen\" at the time when that
    dataset is generated and published, which can make comparisons with datasets generated
    under successive nomenclature conventions difficult.\n<br> \n<br> \n<i>Variation
    in Data Management Standards</i>\n<br> \nThe lack of clear standards with regard
    to the manner in which immunogenetic data are recorded and stored often makes
    it difficult to integrate datasets for meta-analyses, or to even share data between
    research groups. This becomes especially problematic with very large datasets,
    as the process of reformatting data is often accomplished by hand.\n<br> \n<br>
    \n<i>Variation in Ambiguity Reduction Methods</i>\n<br> \nWhen polymorphisms that
    distinguish alleles are not assessed (e.g., because they are in an exon that is
    not interrogated by the typing method employed), the result is allelic ambiguity,
    where exact identity of one or both of the alleles present in a given sample at
    a given locus cannot be known. When it is not possible to establish phase between
    key polymorphisms common to many alleles, the result is genotypic ambiguity, where
    multiple genotypes are possible for a given sample. The choice of one allele over
    another, or of one genotype over another, is the process of ambiguity reduction.
    Currently, there is no standard method for reducing these ambiguities, and different
    research groups may apply different methods to the same set of ambiguities, resulting
    in different alleles and genotypes being chosen for the same typing result.\n<br>
    \n<br> \n<i>High Polymorphism</i>\n<br> \nThe high level of polymorphism associated
    with these genetic systems presents particular bioinformatic, statistical and
    computational challenges that have yet to be addressed in a standardized manner.
    For example, haplotype estimation, case-control association studies and test of
    fit to HWE are all subject to biases introduced by large numbers of low-frequency
    alleles (aka, sparse cells). Guidelines based on theoretical and empirical considerations
    are required for each method in order to insure consistency in the application
    and interpretation of these analyses.\n<br> \n<br> \n<font size=\"+1\"><p style=\"color:black;\"><b>SOLUTIONS
    FOR IMMUNOGENETIC DATA MANAGEMENT AND ANALYSIS</b></p></font>\nGiven these challenges
    to consistent data-analysis, the Immunogenomics Data-Analysis Working Group proposes
    to develop data equivalency standards intended to foster consistency in the use
    of extant and future analytical methods, and to develop novel statistical and
    computational methodologies for the analysis of highly polymorphic loci.\n<br>
    \n<br> \nIn addition, we will determine the impact of various standards and methods
    for data management on downstream data-analyses, comparing them to extant immunogenetic
    data analysis systems, and producing recommendations for consistency in the analysis
    of highly polymorphic datasets.\n<br> \n<br> \nFinally, we will promote widespread
    accessibility and application of these novel data equivalency and analytical tools
    by making them available to the community using web-based and multi-platform approaches.\n\n"
  button:
    enable: false
    label: View Works
    link: project/
service:
  enable: false
  title: Challenges To Consistency In Immunogenomic Data Management And Analysis
  service_item:
  - icon: ''
    name: Variation in Typing Methodology
    content: <div align="left">The large variety of typing methods in use results
      in datasets being generated with varying levels of resolution, so that a particular
      allele (or set of equivalent alleles) may be identified in a sample using one
      method, while a slightly different allele (or set of equivalent alleles) may
      be identified in the same sample, using a different method.</div>
  - icon: ''
    name: Changes in Nomenclature
    content: The continual evolution of the nomenclature conventions over time has
      resulted in a progression of allele and locus identifiers that are not always
      easily inter-related. It is usually the case that the nomenclature used to identify
      the alleles in a dataset is "frozen" at the time when that dataset is generated
      and published, which can make comparisons with datasets generated under successive
      nomenclature conventions difficult.
  - icon: ''
    name: Variation in Data Management Standards
    content: The lack of clear standards with regard to the manner in which immunogenetic
      data are recorded and stored often makes it difficult to integrate datasets
      for meta-analyses, or to even share data between research groups. This becomes
      especially problematic with very large datasets, as the process of reformatting
      data is often accomplished by hand.
  - icon: ''
    name: Variation in Ambiguity Reduction Methods
    content: When polymorphisms that distinguish alleles are not assessed (e.g., because
      they are in an exon that is not interrogated by the typing method employed),
      the result is allelic ambiguity, where exact identity of one or both of the
      alleles present in a given sample at a given locus cannot be known. When it
      is not possible to establish phase between key polymorphisms common to many
      alleles, the result is genotypic ambiguity, where multiple genotypes are possible
      for a given sample. The choice of one allele over another, or of one genotype
      over another, is the process of ambiguity reduction. Currently, there is no
      standard method for reducing these ambiguities, and different research groups
      may apply different methods to the same set of ambiguities, resulting in different
      alleles and genotypes being chosen for the same typing result.
  - icon: ''
    name: High Polymorphism
    content: The high level of polymorphism associated with these genetic systems
      presents particular bioinformatic, statistical and computational challenges
      that have yet to be addressed in a standardized manner. For example, haplotype
      estimation, case-control association studies and test of fit to HWE are all
      subject to biases introduced by large numbers of low-frequency alleles (aka,
      sparse cells). Guidelines based on theoretical and empirical considerations
      are required for each method in order to insure consistency in the application
      and interpretation of these analyses.
cta:
  enable: true
  bg_image: images/call-to-action-bg.jpg
  title: Solutions for Immunogenomic Data Management and Analysis
  content: |-
    <div align="left">
    Given these challenges to consistent data-analysis, the Immunogenomics Data-Analysis Working Group proposes to develop data equivalency standards intended to foster consistency in the use of extant and future analytical methods, and to develop novel statistical and computational methodologies for the analysis of highly polymorphic loci.
    <br />
    <br />
    In addition, we will determine the impact of various standards and methods for data mangement on downstream data-analyses, comparing them to extant immunogenetic data analysis systems, and producing recommendations for consistency in the analysis of highly polymorphic datasets.
    <br />
    <br />
    Finally, we will promote widespread accessibility and application of these novel data equivalency and analytical tools by making them available to the community using web-based and multi-platform approaches.
    </div>
  button:
    enable: false
    label: Tell Us Your Story
    link: contact/
funfacts:
  enable: false
  title: Fun Facts About Us
  description: "'Far far away, behind the word mountains, far from the countries Vokalia
    and Consonantia, <br> there live the blind texts. Separated they live in Bookmarksgrove
    right at the coast of the Semantics'"
  funfact_item:
  - icon: fas fa-mug-hot
    name: Cups Of Coffee
    count: "99"
  - icon: fas fa-glasses
    name: Article Written
    count: "45"
  - icon: fas fa-keyboard
    name: Projects Completed
    count: "125"
  - icon: fas fa-clock
    name: Combined Projects
    count: "200"
  testimonial_slider:
  - name: Raymond Roy
    image: images/clients/avater-1.jpg
    designation: CEO-Themefisher
    content: This Company created an e-commerce site with the tools to make our business
      a success, with innovative ideas we feel that our site has unique elements that
      make us stand out from the crowd.
  - name: Randi Renin
    image: images/clients/avater-1.jpg
    designation: CEO-Themefisher
    content: This Company created an e-commerce site with the tools to make our business
      a success, with innovative ideas we feel that our site has unique elements that
      make us stand out from the crowd.
  - name: Rose Rio
    image: images/clients/avater-3.jpg
    designation: CEO-Themefisher
    content: This Company created an e-commerce site with the tools to make our business
      a success, with innovative ideas we feel that our site has unique elements that
      make us stand out from the crowd.
title: Immunogenomics Data Analysis Working Group
description: Immunogenomics Data Analysis Working Group
bg_image: "/images/blog/blog-post-1.jpg"

---
